testData/preferences/otis.json5

```diff
@@ -0,0 +1,758 @@
+// # otis.json5
+// Test user for Office UI customisations.
+{
+    "flat": {
+        "contexts": {
+            "gpii-default": {
+                "name": "Default preferences",
+                "preferences": {
+                    "http://registry.gpii.net/applications/com.microsoft.office": {
+                        "tabletMode":  true,
+                        "word-ribbon": {
+                            "xmlns$mso": "http://schemas.microsoft.com/office/2009/07/customui",
+                            "mso$ribbon": {
+                                "mso$qat": {
+                                    "mso$sharedControls": {
+                                        "mso$control": [
+                                            {
+                                                "idQ": "mso:FileNewDefault",
+                                                "visible": "false"
```

Comment:
Yowch - are we not planning to ontologise this in any more detailed way?

Reply:
Well, it did cross my mind. Feel free to insist.

Part of the request was "Treat as data block". With that, I think it should go the opposite direction and just point to a blob of data stored somewhere.

Fortunately (or unfortunately?), this blob just so happens to be in a reasonable format where it's technically able to be warped into a preference with no code change, while enjoying blissful ignorance of the actual data structure.

While it's not pretty, it's the most economical solution.

(I can't defend it any further)

Reply:
Our conversations on how to handle such data blocks are still evolving, so let's see how that pans out